CMakeLists_files.cmake

```diff
@@ -113,6 +113,7 @@ list (APPEND EXAMPLE_SOURCE_FILES
   examples/flow_reorder.cpp
   examples/flow_sequential.cpp
   examples/flow_ebos.cpp
+  examples/flow_ebos_2p.cpp
```

Comment:
I suggest to add it to the next list below also, of programs that will be installed.